### ISSUE TITLE
Mouse brain analysis

### ISSUE THREAD

**Issue by araikes** [pnlbwh/TBSS#46]
Hi @tashrifbillah,

I have a couple of questions about this software. I'm looking at using it for mouse brain analyses as I believe it will fill a necessary gap in our research pipeline. To this end, I have a few questions:

1. Is the singularity image still functional (last edit was ~14 months ago) so I can run this on our HPC? Any specific installation instructions?
2. What search rule mask is used in the skeleton projection for the study-specific template particularly if I'm using mouse brains?

Thanks

**Comment by tashrifbillah** [comment]
Hi @araikes ,

> Is the singularity image still functional

It should be, I don't remember any significant correction to this project since then. But if you have a local FSL, you might be better off installing Python 3 + this project. I made things easy for people (I think :p)

> What search rule mask is used in the skeleton projection

We use FSL `tbss_skeleton` script so whatever rule they devised.

**Comment by araikes** [comment]
Hi @tashrifbillah,

Thanks for the responses. 

1. I tried building the singularity image locally and it crashed on the patching step for FSL (line 43). I edited to update to 6.0.3 and skip the patching. I'll see if that solves it.
2. When projecting the all_FA file onto the mean skeleton, the search_rule_mask is the LowerCingulum_1mm standard file for FSL. Any thoughts on the appropriateness here (I'm assuming generally not appropriate).
